src/components/Placeholder/index.tsx

```diff
@@ -1,56 +1,40 @@
 import React, { ReactElement } from 'react';
 import clsx from 'clsx';
-import Icon from '../Icon';
-import Typography from '../Typography';
-import styles from './styles.scss';
+import PlaceholderGraphic from './components/PlaceholderGraphic';
+import PlaceholderTitle from './components/PlaceholderTitle';
+import PlaceholderContent from './components/PlaceholderContent';
+import PlaceholderPrimaryAction from './components/PlaceholderPrimaryAction';
+import PlaceholderSecondaryAction from './components/PlaceholderSecondaryAction';
 import { Props } from './types';
 
 
 function Placeholder({
     className,
-    graphic,
-    title,
-    description,
     children,
 }: Props): ReactElement {
 
 
     const placeholderClasses = clsx(
-        styles.wrapper,
+        'brew-Placeholder',
         className,
     );
 
 
     return (
         <div className={placeholderClasses}>
```

Comment:
This component would be completely acceptable like this.  However, for an even better developer experience you might consider adding the option to pass in the title, graphic, description, etc. _or_ using the components separately.  You could check if any of the title, graphic, etc. props were set and display it one way.  If they aren't set, you can just render children.  Just a thought.